deployment/ansible/otp.yml

```diff
@@ -7,5 +7,4 @@
       apt: update_cache=yes
 
   roles:
-    - { role: "cac-tripplanner.otp"}
```

Comment:
Makes sense. I don't think its worth the duplication since the role dependencies are still explicitly stated in the role itself.